templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -1,10 +1,15 @@
 {
   "Comment": "${step_function_name}: A State machine to ingest DR2 BagIt-like packages into Preservica.",
-  "StartAt": "Map metadata",
+  "StartAt": "Get metadata from JSON & Discovery and update Files table",
   "States": {
-    "Map metadata": {
+    "Get metadata from JSON & Discovery and update Files table": {
```

Comment:
Can we shorten this a bit please. It gets quite hard to parse in the console view of the step functions with long names. "Get metadata and update Files table" is ok.